packages/ros2-test/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ cmake_minimum_required(VERSION 3.5.0)
 
 project(soss-ros2-test)
 
+find_package(statistics_msgs REQUIRED)
+find_package(libstatistics_collector REQUIRED)
+find_package(rosgraph_msgs REQUIRED)
+
```

Comment:
anything that uses `soss_rosidl_mix()` needs these 3 dependencies. We should put these lines upstream but I'm not sure why we need them in the first place if they're unused?

Here's a sample error message:
```
CMake Error at /home/ddeng/traffic_nego_ws/install/soss-core/lib/cmake/soss-core/cmake/soss_mix_generator.cmake:379 (add_library):
  Target "soss-rosidl-ros2-nav_msgs-mix" links to target
  "rosgraph_msgs::rosgraph_msgs__rosidl_generator_c" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
```